tests/plugins/save/save.js

```diff
@@ -1,26 +1,35 @@
 /* bender-tags: editor */
-/* bender-ckeditor-plugins: toolbar,save,wysiwygarea */
+/* bender-ckeditor-plugins: toolbar,save,wysiwygarea,sourcearea */
 
-bender.test( {
-	'test save event': function() {
-		var editor = CKEDITOR.replace( 'editor' ),
-			count = 0;
+function saveTest( editor ) {
+	var count = 0;
 
-		editor.on( 'instanceReady', function() {
-			editor.execCommand( 'save' );
+	editor.on( 'instanceReady', function() {
+		editor.execCommand( 'save' );
 
-			setTimeout( function() {
-				resume( function() {
-					assert.areSame( 1, count, 'save was fired once' );
-				} );
+		setTimeout( function() {
```

Comment:
I wonder if `afterCommandExec` could be used here instead of `setTimeout`.

Reply:
unfortunately no, save plugin doesn't fire `afterCommandExec`